aspnet/config-builder.md

```diff
@@ -210,22 +229,19 @@ The secrets file has the following format:
 
 ```xml
 <add name="AzureKeyVault"
-    [mode|prefix|stripPrefix|tokenPattern]
-    (vaultName="MyVaultName" |
-     uri="https:/MyVaultName.vault.azure.net")
-    [connectionString="connection string"]
-    [version="secrets version"]
-    [preloadSecretNames="true"]
-    type="Microsoft.Configuration.ConfigurationBuilders.AzureKeyVaultConfigBuilder,
-    Microsoft.Configuration.ConfigurationBuilders.Azure" />
+    [mode|@prefix|@stripPrefix|tokenPattern|@escapeExpandedValues|@optional=false]
+    (@vaultName="MyVaultName" | @uri="https://MyVaultName.vault.azure.net")
+    [@version="secrets version"]
+    [@preloadSecretNames="true"]
+    type="Microsoft.Configuration.ConfigurationBuilders.AzureKeyVaultConfigBuilder, Microsoft.Configuration.ConfigurationBuilders.Azure" />
 ```
 
 The [AzureKeyVaultConfigBuilder](https://www.nuget.org/packages/Microsoft.Configuration.ConfigurationBuilders.Azure/) reads values stored in the [Azure Key Vault](/azure/key-vault/key-vault-whatis).
 
-`vaultName` is required (either the name of the vault or a URI to the vault). The other attributes allow control about which vault to connect to, but are only necessary if the application is not running in an environment that works with `Microsoft.Azure.Services.AppAuthentication`. The Azure Services Authentication library is used to automatically pick up connection information from the execution environment if possible. You can override automatically pick up of connection information by providing a connection string.
+`vaultName` is required (either the name of the vault or a URI to the vault). The other attributes allow control about which vault to connect to, but are only necessary if the application is not running in an environment that works with `Microsoft.Azure.Services.AppAuthentication`. The Azure Services Authentication library is used to automatically pick up connection information from the execution environment if possible. You can override automatically pick up of connection information by providing a connection string.  Previous iterations of this config builder allowed for a `connectionString` as a way to supply credential information for connecting to Azure Key Vault. This method is no longer allowed as it is not a supported scenario for the current `Azure.Identity` SDK which is used for connecting to Azure services. Instead, this iteration of the config builder exclusively uses [DefaultAzureCredential](https://docs.microsoft.com/en-us/dotnet/api/azure.identity.defaultazurecredential) from the `Azure.Identity` package to handle credentials for connecting to Azure Key Vault.
```

Comment:
This part is not true anymore. It can be removed - "The other attributes allow control about which vault to connect to, but are only necessary if the application is not running in an environment that works with `Microsoft.Azure.Services.AppAuthentication`. The Azure Services Authentication library is used to automatically pick up connection information from the execution environment if possible. You can override automatically pick up of connection information by providing a connection string."

The added part about obsoleting connection string is good, but feels weird in a bullet point about 'vaultName'.